ocaml-lsp-server/src/client.ml

```diff
@@ -1,33 +1,45 @@
 open Import
 
+module Experimental_capabilities = struct
+  type t = bool
+
+  let of_opt_json (json : Json.t option) =
+    match json with
+    | Some (`Assoc fields) ->
+      Json.field fields "jumpToNextHole" Json.Conv.bool_of_yojson
+      |> Option.value ~default:false
+    | _ -> false
```

Comment:
Seems a bit lax to me. We should error out if this is anything other than `Some Assoc` or `None`.

Reply:
Crashing lsp seems somewhat drastic? I would imagine sending a showMessage notification saying that the experimental capabilities are incorrect in another PR?